esp_lcd_touch_xpt2046.c

```diff
@@ -39,6 +56,8 @@ enum xpt2046_registers
 #define XPT2046_UNLOCK(lock)
 #endif
 
+#define XPT2046_TEMP0_COUNTS_AT_25C   (599.5 / 2507 * 4096)
```

Comment:
If you can convert this line to:
```
static const float XPT2046_TEMP0_COUNTS_AT_25C = (599.5 / 2507 * XPT2046_ADC_LIMIT);
```
And move it below the definition of `XPT2046_ADC_LIMIT` I think we will be good to merge.

Reply:
also if possible add a comment of where in the datasheet this formula came from.

Reply:
On it. Should I bump the version or leave that to you?

Reply:
Sure, let's make it 1.0.4 and if you can add a summary to the changelog.md file. After merge I'll tag 1.0.4 and see how people fare with it :)

Reply:
Version bumped and changelog updated.

Reply:
Looks good, thanks!